applications/DamApplication/custom_strategies/schemes/dam_P_scheme.hpp

```diff
@@ -366,7 +366,7 @@ class DamPScheme : public Scheme<TSparseSpace,TDenseSpace>
         Condition::Pointer rCurrentCondition,
```

Comment:
```suggestion
        Condition& rCurrentCondition,
```

the scheme functions also changed, but you can make that change also in a followup PR